tests/test_get_dem.py

```diff
@@ -92,6 +92,20 @@ def test_get_best_dem_antimeridian():
     assert len(wkt_list) == 241
 
 
+def test_get_best_dem_antimeridian_shifted():
+    # alleutian islands
+    (name, projection, tile_list, wkt_list) = get_best_dem(y_min=51.3, y_max=51.7, x_min=179.5, x_max=180.5)
+    assert name == 'SRTMGL1'
+    assert projection == 4326
+    assert tile_list == ['N51E179', 'N51W180']
+    assert wkt_list == [
+        'POLYGON ((178.999861 52.000139,180.000138777786 52.000139,180.000138777786 50.9998612222142,'
```

Comment:
Unless I'm mistaken, the SAR longitude metadata will not have any values >180 or less than -180.
So, it'll go from 179.9999 to -179.99999, not to 180.00001.  That is, unless we are already dealing with that somewhere?  I don't recall right now.

Reply:
This was Rudi's strategy for dealing with the antimeridian.  `getSubSwath.get_bounding_box_file()` fiddles with the longitude values for granules over the antimeridian: https://github.com/ASFHyP3/hyp3-lib/blob/develop/hyp3lib/getSubSwath.py#L26

So coordinates are shifted from, say `+177` to `-178` into `+177` to `+182`.  The coverage shapefiles are padded out to ~+187, so `get_best_dem` con do the coverage checks for the shifted coordinates, and `write_vrt()` handles putting the right tiles in the right places for doing the eventual `gdal.Warp()` subset.

Reply:
Writing these tests are a way to document the existing behavior.  If there are any users expecting this behavior from `get_dem` or `get_best_dem` we'll be able to detect if we break it as we make improvements going forward.